python/fusion_engine_client/applications/p1_print.py

```diff
@@ -182,6 +187,11 @@ def main():
     # Parse the time range.
     time_range = TimeRange.parse(options.time, absolute=options.absolute_time)
 
+    if options.source_identifier is None:
+        source_ids = [0]
```

Comment:
Please go through carefully and remove this type of thing since it shows up a lot. We should use `None` by default everywhere and display everything, not assume `[0]` is right.